static/js/app.js

```diff
@@ -33,12 +33,20 @@ require('../dist/reporting-views');
           templateUrl: '/partials/analytics.html',
           controller: 'AnalyticsCtrl'
         })
+        .when('/analytics/reporting/:form', {
+          templateUrl: '/partials/analytics.html',
+          controller: 'AnalyticsCtrl'
+        })
+        .when('/analytics/reporting/:form/:facility', {
+          templateUrl: '/partials/analytics.html',
+          controller: 'AnalyticsCtrl'
+        })
```

Comment:
I feel like each URL should resolve to a different controller but it's fine for now.